core/trino-main/src/main/resources/webapp/src/utils.js

```diff
@@ -130,10 +130,13 @@ export function getHumanReadableState(query: any, forOverviewPage: boolean): str
                 if (query.errorCode.name === "USER_CANCELED") {
                     errorMsg = "USER CANCELED";
                 }
+                break;
             case "INTERNAL_ERROR":
                 errorMsg = "INTERNAL ERROR";
+                break;
             case "INSUFFICIENT_RESOURCES":
                 errorMsg = "INSUFFICIENT RESOURCES";
+                break;
             case "EXTERNAL":
                 errorMsg = "EXTERNAL ERROR";
```

Comment:
How about adding `break;` to here as well? Otherwise, we may miss when adding another condition after `EXTERNAL` in the future.

Reply:
Added.